twatbot.py

```diff
@@ -491,35 +496,46 @@ def fetch_more_tweets_callback(user, twats):
 		if in_twatlist(user, twat): return False
 	return True
 
+def get_timestamp(date_format, date=None):
+	if not date: date = time.time()
+	return time.strftime(date_format, time.gmtime(date))
+
 def scrape(user, first_run = False):
 
-	if first_run and not args.count == -2:
+	if first_run and (args.count != -2 and not os.path.isfile(paths.get_user_filename(user))):
```

Comment:
i think "get_user_filename" is a bit misleading. imo it should either have "json" or "db" in the name